src/engine/remotes/PubsubRemotes.ts

```diff
@@ -13,7 +12,7 @@ import { Future, MsgPack, Stopwatch, toJSON } from '../util';
 import { delay, first, ignoreElements, map, reduce, tap, timeout, toArray } from 'rxjs/operators';
 import { MeldError, MeldErrorStatus } from '../MeldError';
 import { AbstractMeld } from '../AbstractMeld';
-import { MeldExtensions, MeldReadState, noTransportSecurity, shortId } from '../../index';
+import { MeldExtensions, MeldReadState, noTransportSecurity, shortId, uuid } from '../../index';
```

Comment:
Noticed that `uuid` is imported from here too `util`.  As before, given that this commit was intended to replace uuid with SHA1, is this actually correct?

Reply:
ditto